apps/charterafrica/src/lib/payload/constants.js

```diff
@@ -0,0 +1,4 @@
+export const ORGANIZATION_COLLECTION = "organisations";
```

Comment:
Put this in `src/payload/utils` or somewhere in `src/payload`: `lib` uses `payload` but `payload` shouldn't depend on `lib`.